build.sbt

```diff
@@ -19,8 +19,8 @@ def projectName(project: sbt.ResolvedProject): String = {
 
 // Provide consistent project name pattern.
 lazy val nameSettings: Seq[Setting[_]] = Seq(
-  normalizedName := projectName(thisProject.value),         // Maven <artifactId>
-  name := s"Scala Native ${projectName(thisProject.value)}" // Maven <name>
+  normalizedName := projectName(thisProject.value), // Maven <artifactId>
+  name := projectName(thisProject.value)            // Maven <name>
```

Comment:
Is this still any different from not having `nameSettings` at all?

Reply:
The `normalizedName` setting was to make sure that all the projects had lowercase names with dashes `-` connecting the words. I did try removing it but then some of the artifacts reverted to other forms. I think a more direct approach of naming the individual projects in `sbt` is more explicit. It is a little more work but could done if you prefer.

Reply:
Ah I see. Then I guess we can keep the setting for `name :=`.

`normalizedName` normally derives from `name`. So I believe that one could be removed.

Reply:
Yes, so this artifact would match the directory.
```scala
lazy val testInterface =
  project
    .in(file("test-interface"))
```
Ok, let me confirm that we can remove.

Reply:
Yes, it is the same before and after the change.
```
sbt:scala-native> testInterface/name
[info] test-interface
sbt:scala-native> testInterface/normalizedName
[info] test-interface
sbt:scala-native> 
```